### DISCUSSION TITLE
Misleading organization and repository names will soon be changed

### DISCUSSION THREAD

**Discussion by ymd-stella** [stella-cv/stella_vslam#267] [Show and tell]
The organization name (OpenVSLAM-Community) and repository name (openvslam) will be changed due to the following issues:

- Some people do not understand that it is a fork, even though it is mentioned in the README.
- Some people have the misconception that the authors of the original version are still maintaining it.

The new organization/repository name will be stella-cv/stella_vslam.

**Comment by ymd-stella** [comment]
Done.

**Comment by Maximvdw** [comment]
Important question, what is the name that we should use to refer to this code base? Still openvslam (as the post seems to refer to only the repo name)?

**Reply by ymd-stella** [reply]
This repository is one of the unofficial implementations of the paper "OpenVSLAM: A Versatile Visual SLAM Framework". If you are going to cite the algorithm, you should refer to the paper. If you want to refer to the implementation in particular, I think you should refer to the repository name (stella_vslam).
